_templates/local-data-story-card/with-prompt/src/components/index.ejs.t

```diff
@@ -0,0 +1,55 @@
+---
+to: packages/<%=package%>/src/components/<%=StoryCardName%>/index.js
+---
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { CivicCard } from "@hackoregon/component-library";
+
+import <%=StoryCardName%>Meta from "./<%=storyCardName%>Meta";
+import { fetch<%=StoryCardName%>Data } from "../../state/<%=slug%>/actions";
```

Comment:
In `_templates/local-data-story-card/with-prompt/src/state/actions.ejs.t` the path where `actions.js` is created is would actually be 
`../../state/<%=slug%>-data/actions;`
not:
`../../state/<%=slug%>/actions;`